typescript/civica/name-record.ts

```diff
@@ -10,8 +10,38 @@ export type CivicaNameRecord = {
   streetAddress: StreetAddress[]
 }
 
-export type StreetAddress = {
+export interface StreetAddress {
   address1: string
   address2: string
   postcode: string
 }
+
+export interface CivicaNameRecordElementConfigurations {
+  givenNameLabel?: string
+  givenNameLabelIsRequired?: boolean
+  givenNameLabelIsHidden?: boolean
```

Comment:
These should prob be `givenName1`

and the required and hidden config, should not have the word the `label` in it.